File: EDoctorsApp/EDoctors/app/src/main/java/com/incubation_lab/edoctors/UserLoginFragment.java
package com.incubation_lab.edoctors;

import android.os.Bundle;
import android.os.Handler;
import android.transition.TransitionManager;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import androidx.annotation.NonNull;
import androidx.constraintlayout.widget.ConstraintLayout;
import androidx.constraintlayout.widget.Placeholder;
import androidx.fragment.app.Fragment;

public class UserLoginFragment extends Fragment {
    public static Fragment newInstance() {
        return new UserLoginFragment();
    }

    private Placeholder logoPlaceHolder;
    private ImageView logo;
    private ConstraintLayout bottomContainer;
    private  ConstraintLayout parent;
    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {

        View root = inflater.inflate(R.layout.fragment_user_login, container, false);
        parent= root.findViewById(R.id.parent_container);
        bottomContainer = root.findViewById(R.id.user_login_bottom_container);
        bottomContainer.setVisibility(View.GONE);
        logo= root.findViewById(R.id.user_login_logo);
        logoPlaceHolder= root.findViewById(R.id.user_login_placeholder);


        Handler handler = new Handler();
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {
                TransitionManager.beginDelayedTransition(parent);
                logoPlaceHolder.setContentId(R.id.user_login_logo);
                bottomContainer.setVisibility(View.VISIBLE);
            }
        },100);




        return root;
    }
}

File: EDoctorsApp/app/src/main/java/com/incubation_lab/edoctor/UserLoginFragment.kt
package com.incubation_lab.edoctor

import android.os.Bundle
import android.os.Handler
import android.transition.TransitionManager
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.incubation_lab.edoctor.databinding.FragmentUserLoginBinding


class UserLoginFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        var binding = FragmentUserLoginBinding.inflate(inflater)
        binding.userLoginBottomContainer.visibility = View.GONE
        val root = binding.root
        Handler().postDelayed(Runnable { TransitionManager.beginDelayedTransition(root)
            binding.userLoginPlaceholder.setContentId(R.id.user_login_logo)
            binding.userLoginBottomContainer.visibility = View.VISIBLE
0                                       },100)





        return root
    }

    companion object {

        @JvmStatic
        fun newInstance() =
                UserLoginFragment()
    }
}